otelopscol/manifest.yaml

```diff
@@ -8,78 +8,77 @@ dist:
   output_path: ./_build
```

Comment:
Looks like this was edited manually. This will need to be properly generated by `distrogen` before merge. The process will be:

* Change the `opentelemetry_version` fields in `specs/otelopscol.yaml` 
* Add the local `transformprocessor` to `components/otelopscol/registry.yaml`

Once you regenerate you should have the same setup as what you've manually done here.

Reply:
Thank you for the comment! Updated `specs/otelopscol.yaml`,  `components/otelopscol/registry.yaml` and ran distrogen. Done :)